test/integration/logbucket-project/logbucket_project_test.go

```diff
@@ -0,0 +1,96 @@
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLogBucketProjectModule(t *testing.T) {
+
+	const logApiFdqm = "logging.googleapis.com"
+	const RETENTION_DAYS = 20
+	const ROLE_BUCKET_WRITER string = "roles/logging.bucketWriter"
+
+	bpt := tft.NewTFBlueprintTest(t,
+		tft.WithTFDir("../../../examples/logbucket/project"),
+	)
+	bpt.DefineVerify(func(assert *assert.Assertions) {
+		bpt.DefaultVerify(assert)
+
+		//*******************************
+		// Bucket on other project test *
+		//*******************************
+		projId := bpt.GetStringOutput("log_bucket_project")
+		bktName := bpt.GetStringOutput("log_bucket_name")
+		bktDestProjId := bpt.GetStringOutput("log_bkt_same_proj")
+		sinkProjId := bpt.GetStringOutput("log_sink_project_id")
+		sinkDest := bpt.GetStringOutput("log_sink_destination_uri")
+		sinkName := bpt.GetStringOutput("log_sink_resource_name")
+		sinkWriterIdentity := bpt.GetStringOutput("log_sink_writer_identity")
+
+		logBucketDetails := gcloud.Runf(t, fmt.Sprintf("logging buckets describe %s --location=%s --project=%s", bktName, "global", projId))
+
+		// assert log bucket name, retention days & location
+		assert.Equal(sinkDest[len(logApiFdqm)+1:], logBucketDetails.Get("name").String(), "log bucket name should match")
+		assert.Equal(int64(30), logBucketDetails.Get("retentionDays").Int(), "retention days should match")
+
+		logSinkDetails := gcloud.Runf(t, fmt.Sprintf("logging sinks describe %s --project=%s", sinkName, sinkProjId))
+
+		// assert log sink name, destination & filter
+		assert.Equal(sinkDest, logSinkDetails.Get("destination").String(), "log sink destination should match")
+		assert.Equal("resource.type = gce_instance", logSinkDetails.Get("filter").String(), "log sink filter should match")
+		assert.Equal(sinkWriterIdentity, logSinkDetails.Get("writerIdentity").String(), "log sink writerIdentity should not be empty")
+
+		//**********************************
+		// Bucket on the same project test *
+		//**********************************
+		sameProjId := bpt.GetStringOutput("log_bkt_same_proj")
+		sameProjBktName := bpt.GetStringOutput("log_bkt_name_same_proj")
+		sameProjSinkProjId := bpt.GetStringOutput("log_sink_id_same_proj")
+		sameProjSinkDest := bpt.GetStringOutput("log_sink_dest_uri_same_proj")
+		sameProjSinkName := bpt.GetStringOutput("log_sink_resource_name_same_proj")
+
+		sameProjBktDetails := gcloud.Runf(t, fmt.Sprintf("logging buckets describe %s --location=%s --project=%s", sameProjBktName, "global", sameProjId))
+
+		// assert log bucket name, retention days & location
+		assert.Equal(sameProjSinkDest[len(logApiFdqm)+1:], sameProjBktDetails.Get("name").String(), "log bucket name should match")
+		assert.Equal(int64(RETENTION_DAYS), sameProjBktDetails.Get("retentionDays").Int(), "retention days should match")
```

Comment:
two diferente values?
```
assert.Equal(int64(30), logBucketDetails.Get("retentionDays").Int(), "retention days should match")
```

Reply:
Created constant defaultRetentionDays and kept both examples with this default for user better understanding code and example